stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -366,12 +366,25 @@ public struct ${Self}<Pointee>
 %  end
 
   /// Rebinds memory at `self` to type `T` with capacity to hold `count`
-  /// adjacent `T` values while executing the `body` closure. After executing
-  /// the closure, rebinds memory back to `Pointee`.
+  /// adjacent `T` values while executing the `body` closure.
+  ///
+  /// After executing the closure, rebinds memory back to `Pointee`.
   ///
   /// - Precondition: Type 'T' is layout compatible with type 'Pointee'.
   /// - Precondition: The memory `self..<self + count * MemoryLayout<T>.stride`
   ///   is bound to `Pointee`.
+  ///
+  /// Accessing an ${Self}<T> requires that the memory be "bound" to type `T`.
```

Comment:
Good point. I changed it to "Accessing `${Self}<T>.pointee` requires..." I think that gets the point across. Note that the doc comment on `subscript` says "Accesses the pointee at `self   + i`.